gitlab/resource_gitlab_project_approval_rule_test.go

```diff
@@ -0,0 +1,416 @@
+package gitlab
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+var gitlabProjectApprovalRuleSkipAttributes = []string{
+	"project", "user_ids", "group_ids", "eligible_approvers",
+}
+
+type checkGitlabProjectApprovalRule struct {
+	SkippedAttributes []string
```

Comment:
What is the purpose of this field if it is always set to `gitlabProjectApprovalRuleSkipAttributes`?